gcp/appengine/frontend3/src/templates/home.html

```diff
@@ -14,7 +14,9 @@ <h1 class="title">A distributed vulnerability database for Open Source</h1>
         <a class="cta-primary link-button" href="{{ url_for('frontend_handlers.list_vulnerabilities') }}">Search
           Vulnerability Database</a>
         <a class="cta-primary link-button" href="#use-the-api">Use the API</a>
-        <a class="cta-primary link-button" href="#use-the-cli">CLI Tools</a>
+        <a class="cta-primary link-button" href="#use-vulnerability-scanner">Vulnerability Scanner</a>
+        <a class="cta-primary link-button" href="#use-remediation-tools">Remediation Tools</a>
+        <a class="cta-primary link-button" href="#use-the-github-action">Github Action Tools</a>
```

Comment:
nit: Can we call this "GitHub Workflows"?

Reply:
♻️ updated via 4d3076cc31c101dde0b31dd7ca54828b04514d94